autopush/endpoint.py

```diff
@@ -558,6 +569,9 @@ def _router_completed(self, response, uaid_data):
                               **self._client_info)
             time_diff = time.time() - self.start_time
             self.metrics.timing("updates.handled", duration=time_diff)
+            if warning:
+                response.response_body += (
+                    " "[:len(response.response_body) > 0] + warning)
```

Comment:
Ok, this is the most creative way I've seen to attach a conditional space. 

:clap: